packages/prisma/migrations/20220330071743_add_dynamic_group_booking/migration.sql

```diff
@@ -0,0 +1,8 @@
+-- AlterTable
+ALTER TABLE "Booking" ADD COLUMN     "dynamicEventSlugRef" TEXT;
+
+-- AlterTable
+ALTER TABLE "Booking" ADD COLUMN     "dynamicGroupSlugRef" TEXT;
+
+-- AlterTable
+ALTER TABLE "users" ADD COLUMN     "allowDynamicBooking" BOOLEAN DEFAULT false;
```

Comment:
We default this setting to false so that only users who explicitly allow it are allowed to be booked through dynamic group booking, or it might be messy for them if they don't visit their profile settings too often.